src/private/logs.ts

```diff
@@ -28,7 +20,13 @@ export namespace logs {
   export function registerGetLogHandler(handler: () => string): void {
     ensureInitialized();
 
-    GlobalVars.getLogHandler = handler;
-    handler && sendMessageRequestToParent('registerHandler', ['log.request']);
+    if (handler) {
+      registerHandler('log.request', () => {
+        const log: string = handler();
+        sendMessageToParent('log.receive', [log]);
+      });
+    } else {
```

Comment:
why remove handler is not passed. Is this for handling 2nd call to override it?

Reply:
I am not sure what you mean by this. I was simply copying the existing logic for logs.